--- ./orig/sysconfig.py	2021-05-29 14:24:44.457014340 +0900
+++ ./python3.8/sysconfig.py	2021-05-29 14:21:13.860319173 +0900
@@ -421,6 +421,9 @@ def _init_posix(vars):
     _temp = __import__(name, globals(), locals(), ['build_time_vars'], 0)
     build_time_vars = _temp.build_time_vars
     vars.update(build_time_vars)
+    with open('/mimic-cross/data/config_vars.pickle', 'rb') as f:
+        import pickle
+        vars.update(pickle.load(f))
 
 
 def _init_non_posix(vars):
